rs/registry/local_registry/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::disallowed_types)]
```

Comment:
Maybe, all of these legacy ignores should say that they are legacy. Otherwise, people should normally include a comment that explains why the ban does not cover their use case. There shouldn't be bare `allows`.

Reply:
I am not sure if it matters it is legacy or not. Maybe the default is if there is no comment explaining why the allow is there then the allow should be fixed and used.